cloud/aws/cli/core.py

```diff
@@ -9,24 +10,34 @@
 import sys
 import io
 
-conf = dynaconf.Dynaconf(
-    load_dotenv=True,
-    envvar_prefix=False,
-    validators=[
-        dynaconf.Validator("VAST_AWS_REGION", must_exist=True),
-        dynaconf.Validator("VAST_CIDR", must_exist=True),
-        dynaconf.Validator("VAST_PEERED_VPC_ID", must_exist=True),
-        dynaconf.Validator("VAST_LAMBDA_IMAGE"),  # usually resolved lazily
-        dynaconf.Validator("VAST_VERSION"),  # usually resolved lazily
-        dynaconf.Validator("VAST_SERVER_STORAGE_TYPE", default="EFS"),
-    ],
-)
+VALIDATORS = [
+    dynaconf.Validator("VAST_AWS_REGION", must_exist=True),
+    dynaconf.Validator("VAST_CIDR", must_exist=True),
+    dynaconf.Validator("VAST_PEERED_VPC_ID", must_exist=True),
+    dynaconf.Validator("VAST_VERSION"),  # usually resolved lazily
+    dynaconf.Validator("VAST_SERVER_STORAGE_TYPE", default="EFS"),
+]
```

Comment:
the extractors are now separate because they will typically depend on the module that is being deployed. For instance when running `cloudtrail.deploy`, it will be the source bucket that will be required.